server/route.js

```diff
@@ -84,6 +87,17 @@ Route = function(router, path, options) {
       res.pushData('fast-render-data', page.frData);
       next();
     }
+
+    function moveScripts(data) {
+      var $ = Cheerio.load(data);
+      var heads = $('head link, head script');
```

Comment:
I think we should only do this for scripts. Stylesheets needs to be render blocking. 
No one needs to defer it, I think.
Otherwise, page will get messy,